src/hooks/use-fetch-wpcom-sites/index.tsx

```diff
@@ -156,7 +141,34 @@ export const useFetchWpComSites = ( connectedSiteIds: number[] ) => {
 				}
 			);
 
-			isInitialized.current = true;
+			const syncSites = transformSiteResponse(
+				response.sites,
+				allConnectedSites.map( ( { id } ) => id )
+			);
```

Comment:
```suggestion
			const syncSites = transformSiteResponse( response.sites, [] );
```
Since we're always running `transformSiteResponse` for `syncSitesWithSyncSupportForSelectedSite` anyway, I believe we could just pass an empty array here, right? It doesn't matter what value we store for `syncSupport` in `appdata`.

Reply:
Maybe this is actually a decent enough simplification to not move the `already-connected` logic elsewhere… Though I still find it very tempting since that's the only non-global sync status.

Reply:
`It doesn't matter what value we store for syncSupport in appdata.` - hm, maybe you are right,  but I want to keep it appdata-v1.json with correct syncStatus, to avoid confusing us in the future with not aligned data between appdata-v1.json and what we have in runtime.